R/deprecated.R

```diff
@@ -59,7 +59,7 @@ diskCache <- function(
   exec_missing = FALSE,
   logfile = NULL)
 {
-  shinyDeprecated("cachem::cache_disk", version = "1.5.1")
+  shinyDeprecated("1.5.1", "diskCache()", "cachem::cache_disk()")
```

Comment:
```suggestion
  shinyDeprecated("1.6.0", "diskCache()", "cachem::cache_disk()")
```

Also, I just realized that it should include a check  for `exec_missing` -- I don't think anyone used that argument, but it doesn't do anything now. It should be a warning, even outside of dev mode. I'm not sure the best way to do this. Maybe something like this?

```R
if (!missing(exec_missing)) {
  warning( ... )
}
```